src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+#pylint: disable=line-too-long
+
+from azure.cli.commands.parameters import (location_type,)
+from azure.cli.commands import register_cli_argument, CliArgumentType
+
+
+name_type = CliArgumentType(
+    options_list=('--name', '-n'),
+    help='Name for new Azure IoT Hub.')
+sku_type = CliArgumentType(
+    choices=['F1', 'S1', 'S2', 'S3'], default='F1',
```

Comment:
>default='F1' [](start = 38, length = 12)

Since you specify the default in your method signature, you don't need to specify it here.